tachyon/zk/expressions/evaluator/simple_selector_extractor_unittest.cc

```diff
@@ -5,26 +5,27 @@
 #include "tachyon/zk/expressions/evaluator/test/evaluator_test.h"
 #include "tachyon/zk/expressions/expression_factory.h"
 
-namespace tachyon::zk {
+namespace tachyon::zk::evaluator {
 
-using Expr = std::unique_ptr<Expression<GF7>>;
-
-class SimpleSelectorExtractorTest : public EvaluatorTest {};
+class SimpleSelectorExtractorTest : public EvaluatorTest {
+ public:
+  using Expr = std::unique_ptr<Expression<GF7>>;
+};
```

Comment:
These change are unneeded and the way to test is different from the previous one in `selector_replacer_unittest.cc`